src/servers/graphql-server.ts

```diff
@@ -84,7 +84,6 @@ const setGqlContext = async (
   res: Response,
   next: NextFunction,
 ): Promise<void> => {
-  console.log(req)
```

Comment:
Add a line here `if (!tokenPayload) next()`

Reply:
this will not record ip in span and also could not pass  logger and other values to public queries/muttions